pan-domain-auth-play/src/main/scala/com/gu/pandomainauth/action/Actions.scala

```diff
@@ -37,6 +38,18 @@ trait AuthActions extends PanDomainAuth {
    */
   def cacheValidation: Boolean = false
 
+
+  /**
+   * Adding an expiry extension to `APIAuthAction`s allows for a delay between an applications authentication and their
+   * respective API XHR calls expiring.
+   *
+   * This is particularly useful for SPAs where users have third party cookies disabled.
+   *
+   * @return the amount of delay between App and API expiry in milliseconds
+   */
+  def apiExpiryExtension: Long = 1.hour.toMillis
+
```

Comment:
Agreed again - leave it to the clients to decide their grace period...
This does however mean that if we're accessing Workflow and the Grid from Composer, one might fall out of grace and the other not.


I don't foresee this being a problem - probably a benefit for people to find their best / most secure grace period.